hydrosdk/cluster.py

```diff
@@ -83,20 +126,26 @@ def servables(self):
     def applications(self):
         return []
 
-    def build_info(self):
+    def build_info(self, manager=True, gateway=True, sonar=True) -> dict:
```

Comment:
Why?

Reply:
By default we have a http-protocol. It uses all of the mentioned services to check its status. An extra is a grpc and it uses only some of them. So the most common option of options to be used is set here

Reply:
> An extra is a grpc and it uses only some of them.
Nope. GRPC doesn't have build_info at all. It's completely different protocol with different API endpoints.

Reply:
And by selecting services for build_info you only use http protocol to make such requests

Reply:
dumped build_info for grpc